spec/ddtrace/contrib/delayed_job/patcher_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+require 'ddtrace'
+require_relative 'active_record_setup'
+
+RSpec.describe Datadog::Contrib::DelayedJob::Patcher, :delayed_job_active_record do
```

Comment:
I dig the use of the trait here to drive test configuration, but I think we still need to update the `active_record_setup.rb` file to match, no?

Reply:
done